### ISSUE TITLE
No Error being thrown in PluginConfig

### ISSUE THREAD

**Issue by NotWaterScriptNhttp** [northwood-studios/NwPluginAPI#202]
I had this issue where my plugin would suddenly break the whole server on launch, it was cause of an error inside my PluginConfig and it took me a while to find why it was happening.
I think this is a bug as every runtime error should be reported to the console.

**Comment by SrLicht** [comment]
Does this happen in the beta of the new version of the API?

**Comment by NotWaterScriptNhttp** [comment]
Yes this still does not throw any error to the console even in the beta.

**Comment by NotWaterScriptNhttp** [comment]
And just to add i have/had debug mode toggled.

**Comment by moddedmcplayer** [comment]
It should be in serverlogs, see #105

**Comment by NotWaterScriptNhttp** [comment]
Yeah its #105 (prob didn't see it or smth), but still there is no error in the ServerLogs.
